config/runtimes/kserve-tensorflow-serving-gpu.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: serving.kserve.io/v1alpha1
+kind: ClusterServingRuntime
+metadata:
+  name: kserve-tensorflow-serving-gpu
+spec:
+  supportedModelTypes:
+    - name: tensorflow
+      version: "1"
```

Comment:
We should also add tensorflow version "2" to the supported list.

Reply:
Version is not specified on the tfserving runtime right? The runtime template can work with both 1.x and 2.x unless 2.x has deprecated some of the runtime arguments used on the template. I think we probably only want to create new runtime if different versions require different configurations.